ocaml/xapi-aux/throttle.mli

```diff
@@ -22,3 +22,33 @@ module Make (_ : SIZE) : sig
 
   val execute : (unit -> 'a) -> 'a
 end
+
+module Batching : sig
+  (** batching delay configuration *)
+  type t
+
+  val make : delay_before:float -> delay_between:float -> t
```

Comment:
There's a distinct lack of units here, please use spans (and I guess keeps the checks when creating the delays to keep anything smaller than epsilon * seconds to be zero); or use seconds here

Reply:
Replaced with spans (although it makes arithmetic a bit more awkward, e.g. there is no division operator for spans).